src/app/routes/activation.ts

```diff
@@ -16,16 +16,29 @@ class ActivationController {
     this.service = service;
   }
 
+  async sendDeactivationEmail(req: Request<{ email: string }>, res: Response) {
+    const email = req.params.email.toLowerCase();
+    try {
+      const user = await this.service.User.FindUserByEmail(email);
+      const uuid = user.uuid;
+      await this.service.User.deactivate(email);
+      res.status(200).send({ error: null, message: 'Email sent' });
+      AnalyticsService.trackDeactivationRequest(uuid, req);
+    } catch (error) {
```

Comment:
you can remove `try-catch` safely unless is required for some custom behavior as we added a catch-all handler to make the code easier to read and in order to centralize error logging. 

Also, please add a log for being able to locate where a user has arrived and where not in the deactivation steps when an error happens:

`logger.info('User %s requested deactivation', email);`